config/locales/pl.yml

```diff
@@ -228,6 +228,7 @@ pl:
     delete_account_html: Jeżeli chcesz usunąć konto, <a href="%{path}">przejdź tutaj</a>. Otrzymasz prośbę o potwierdzenie.
     didnt_get_confirmation: Nie otrzymałeś instrukcji weryfikacji?
     forgot_password: Nie pamiętasz hasła?
+    invalid_reset_password_token: Link resetowania hasła jest nieważny lub nieprawidłowy. Spróbuj ponownie.
```

Comment:
What about the small issue with missing “do”? Actually, since the real issue is not with the link (an URL may be even copy-pasted from an email without a link), but with a token, I would make it „Token resetowania hasła jest nieważny lub nieprawidłowy” — if we’re not obligated to translate from English…